### ISSUE TITLE
Make field visible on condition

### ISSUE THREAD

**Issue by ScottB-Ronco** [hougaard/Simple-Object-Designer#240]
I would like to know if it is possible with SOD to have field made visible or editable based on another field.

Right now I am creating a new feature called Vendor Information Card in this feature I am added a boolean field (toggle field) and a date field.

Want I want to achieve is that when the boolean field is toggled on then either show date field or allow it to be modified. If toggle is off clear date field and make it read only or hide it.

I suspect this isn't something SOD can do, but I figured I would still ask.

If I am right and it can't be done, could this be a possible feature on a future release?

**Comment by hougaard** [comment]
You're right, this isn't something you can do with the Simple Object Designer, it's not "simple" because Visible/editable are controlled by client-side expressions, that are really limited...

Maybe in the future :)
